modular_skyrat/modules/customization/modules/language/nekomimetic.dm

```diff
@@ -0,0 +1,3 @@
+// Nekomimetic, my beloved
+/datum/language/nekomimetic
+	secret = TRUE
```

Comment:
this is a language used by an actively selectable species(felinids), this needs to be selectable